src/GridFS/StreamWrapper.php

```diff
@@ -296,8 +324,8 @@ private function initReadableStream(): bool
 
         assert($this->protocol !== null);
         $this->stream = new ReadableStream(
-            $context[$this->protocol]['collectionWrapper'],
-            $context[$this->protocol]['file'],
+            $this->contextOptions['collectionWrapper'],
```

Comment:
The first two lines of this method with the `assert()` and `stream_context_get_options()` call no longer seem relevant. Likewise for `initWritableStream()` below.